README.md

```diff
@@ -56,6 +56,7 @@ An ([IDE](https://en.wikipedia.org/wiki/Integrated_development_environment)) is
 - [HASH](https://hash.ai) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Build and run JavaScript, Python, or Rust multi-agent simulations in the browser. Provides visualization, analysis, and experiment features.
 - [goormIDE](https://ide.goorm.io) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run almost every programming languages with real-time collaboration, live pair programming, and built-in messenger. 
 - [Coding Rooms](https://codingrooms.com/compiler/) - Edit and run JavaScript, Java, Python, C++ and more in a collaborative environment with intellisense, live chat, audio and video conferencing, real-time editing, and terminal.
+- [Dockside](https://dockside.io/) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/newsnowlabs/dockside) [<img title="Self Hosted" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/docker.svg" />](https://hub.docker.com/r/newsnowlabs/dockside) - Run parallel access-controlled development containers, from vanilla or production environment images, featuring the Theia IDE (complete with terminals) for developing in any language for multiple architectures, within choice of Docker runtimes; stage running code publicly or securely on own domains over HTTPS.
```

Comment:
```suggestion
- [Dockside](https://dockside.io/) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/newsnowlabs/dockside) [<img title="Self Hosted" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/docker.svg" />](https://hub.docker.com/r/newsnowlabs/dockside) - Self host parallel access-controlled development containers, from vanilla or production environment images, featuring the Theia IDE (complete with terminals) for developing in any language for multiple architectures, within choice of Docker runtimes; stage running code publicly or securely on own domains over HTTPS.
```

Reply:
Thanks @styfle that's excellent, really happy with this.

Reply:
If you apply that change then we can go ahead and merge, thanks!